From 9f5754c142003d69fa6caebae6a0bcb85b2e2e2e Mon Sep 17 00:00:00 2001
From: WofWca <wofwca@protonmail.com>
Date: Sun, 24 May 2026 20:36:06 +0400
Subject: [PATCH] improvement: lower blood bounce mark min speed

We have increaset `MAX_MARK_POLYS` and basically lowered
the bounce factor so now it is safe and is better-looking
to require less speed to leave a blood mark.
---
 code/cgame/cg_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/code/cgame/cg_main.c b/code/cgame/cg_main.c
index ef40256c..28ed3cfb 100644
--- a/code/cgame/cg_main.c
+++ b/code/cgame/cg_main.c
@@ -256,7 +256,7 @@ static cvarTable_t cvarTable[] = {
 	{ &cg_simpleItems, "cg_simpleItems", "0", CVAR_ARCHIVE },
 	{ &cg_addMarks, "cg_marks", "1", CVAR_ARCHIVE },
 	// Note that ~290 corresponds to a free fall with no bounce from player height.
-	{ &cg_bounceMarksMinImpactSpeed, "cg_bounceMarksMinImpactSpeed", "350", CVAR_ARCHIVE },
+	{ &cg_bounceMarksMinImpactSpeed, "cg_bounceMarksMinImpactSpeed", "200", CVAR_ARCHIVE },
 	{ &cg_bounceSoundMinImpactSpeed, "cg_bounceSoundMinImpactSpeed", "450", CVAR_ARCHIVE },
 	{ &cg_lagometer, "cg_lagometer", "1", CVAR_ARCHIVE },
 	{ &cg_railTrailTime, "cg_railTrailTime", "400", CVAR_ARCHIVE  },
-- 
2.55.0

